sdk/python/.pylintrc

```diff
@@ -182,7 +182,7 @@ contextmanager-decorators=contextlib.contextmanager
 # List of members which are set dynamically and missed by pylint inference
 # system, and so shouldn't trigger E1101 when accessed. Python regular
 # expressions are accepted.
-generated-members=empty_pb2.Empty,struct_pb2.Struct,struct_pb2.ListValue
+generated-members=empty_pb2.Empty,struct_pb2.Struct,struct_pb2.ListValue,resource_pb2.*,provider_pb2.*,engine_pb2.*,language_pb2.*,plugin_pb2.*,proto.*
```

Comment:
everything is dynamic now and pylint doesn't look at the type hints in the pyi files